Scripts/prettyprint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+function print_usage_and_exit()
+{
+    echo "Usage: $0 output_dir [input_files]"
+    exit 1
+}
+
+# Set up directories
+SCRIPTS_DIR=$(cd "$(dirname "$0")" && pwd)
+TESTS_DIR="$SCRIPTS_DIR/../Tests"
+SUITE_TEST_DIR="$TESTS_DIR/JSONSchemaTestSuite/tests/draft4"
+CUSTOM_TEST_DIR="$TESTS_DIR/JSONSchemaCustom"
+
+if [ -z "$PRETTYPRINTER_PATH" ]; then
+    PRETTYPRINTER_PATH="/Users/jillcohen/Library/Developer/Xcode/DerivedData/TWTValidation-hhdpirveagehwtciiqnlfjuvcrrx/Build/Products/Debug/JSONSchemaPrettyPrinter"
```

Comment:
We probably need to figure this out a little better. Maybe just add a comment that says how a user should find the appropriate path for this.